CRM/Price/DAO/PriceSet.php

```diff
@@ -382,11 +382,15 @@ public static function &fields() {
         ],
         'min_amount' => [
           'name' => 'min_amount',
-          'type' => CRM_Utils_Type::T_INT,
+          'type' => CRM_Utils_Type::T_MONEY,
```

Comment:
This is correct.. but we should test to make sure it doesn't mess up any formatting on pages that were treating this as an int.